app/templates/README.mdown

```diff
@@ -69,3 +69,67 @@ Or on Windows:
 The application comes with a sample set of JSON data. The detail controller,
 the part that handles showing your data, can not only handle JSON, but also
 XML, Binary, and Text data out of the box.
+
+## Running with PM2
+
+To run you must install pm2 with:
+
+`npm install pm2 -g`
+
+Make sure you have the file `process.json` if it's missing then run:
+
+`gulp init-process`
+
+This file will also include any config your node layer needs eg. mlHost, mlPort, nodePort, mlPassword etc.
+
+You can then launch the app using pm2 with:
+
+`pm2 startOrRestart process.json`
+
+You can also launch the app using different environment configuration (env name prefixed with "env_") using:
+
+`pm2 startOrRestart process.json --env build`
+
+To view all running pm2 applications use:
+
+`pm2 list`
```

Comment:
See above, run as pm2 users.. (mention pm2 kill if you accidentally launch it under your own account). Or should we mention `sudo -u pm2 pm2 list` instead to prevent mistakes?